embeds/connectors/outbound/graphite.md

```diff
@@ -1,21 +1,16 @@
-# InfinyOn Graphite Sink Connector
-The [Graphite][4] Sink connector reads records from Fluvio topic and sends them to
-the configured Graphite's Metric using the PlainText approach.
+# graphite-sink
+Graphite Metrics Server Fluvio Sink Connector
 
-# Configuration
+## Usage
```

Comment:
we should update this readme to match all the added information that is currently on main